### PR TITLE
Fixed activties page and services

### PR DESCRIPTION
**Pull request by panchoarc**
## Summary 

- Fixed Activities services
- Fixed problems with files.

### CODE REVIEW

File: src/controllers/activities.js [hunk 1]
```diff
@@ -16,13 +21,14 @@ const postActivities = async function (req, res, next) {
...
       const { url } = await uploadFile(req.file, next);
+      console.log(url);
       req.body.image = url;
     } else {
       req.body.image = null;
     }
 
     const newActivity = await db[entity].create(req.body);
-    res.status(201).send({
+    return res.status(201).send({
```
Comment (rafalynch) [inline src/controllers/activities.js]:
```suggestion
    res.status(201).send({
```

File: src/controllers/files.js [hunk 2]
```diff
@@ -10,6 +10,7 @@ const getFileByKey = async function (req, res, next) {
   try {
     const key = req.params.key;
     const readStream = await downloadFile(key, next);
+    console.log(readStream);
```
Comment (rafalynch) [inline src/controllers/files.js]:
```suggestion

```

File: src/controllers/activities.js [hunk 3]
```diff
@@ -16,13 +21,14 @@ const postActivities = async function (req, res, next) {
   try {
     if (req.file) {
       const { url } = await uploadFile(req.file, next);
+      console.log(url);
```
Comment (rafalynch) [inline src/controllers/activities.js]:
```suggestion

```

File: src/services/aws_s3.js [hunk 4]
```diff
@@ -12,93 +14,91 @@ const region = process.env.AWS_REGION;
...
-  };
-    
-    const object =  await s3.putObject(uploadParams)
-    const url = generateS3Url(key)
-    return {object, url}
-  }
-  catch(err){
-    next(err)
+  try {
+    console.log(file);
```
Comment (rafalynch) [inline src/services/aws_s3.js]:
```suggestion

```

**Review by rafalynch — CHANGES_REQUESTED** [review]

**Comment by panchoarc** [comment]
Cambios realizados, probar y aprobar

**Pull request closed without merging**